addon/mixins/application-route-mixin.js

```diff
@@ -15,20 +15,22 @@ const { inject, on } = Ember;
 
   ```js
   // app/instance-initializers/session-events.js
-  Ember.Application.initializer({
-    name:       'session-events',
-    after:      'ember-simple-auth',
-    initialize: function(container, application) {
-      var applicationRoute = container.lookup('route:application');
-      var session          = container.lookup('service:session');
+  export function initialize(instance) {
+      var applicationRoute = instance.container.lookup('route:application');
+      var session          = instance.container.lookup('service:session');
       session.on('authenticationSucceeded', function() {
         applicationRoute.transitionTo('index');
       });
       session.on('invalidationSucceeded', function() {
         window.location.reload();
       });
-    }
   });
+
+  export default {
+    initialize,
+    name:    'session-events',
+    after:   'ember-simple-auth',
```

Comment:
Your own example you just gave included that comma, so I thought it was from some style guide or something. I'll remove the spaces for you. -_-